src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -1870,6 +1870,14 @@ private void handleBulkResourceBoxMigration(String sessionId,
 					{
 						allSiteOwners.append(",").append(userEmail);
 					}
+					else if (siteId.startsWith(Utils.CTOOLS_SITE_TYPE_MYWORKSPACE_PREFIX))
```

Comment:
This section about appending the userEmail will, IMHO, be a lot clearer and easier to maintain if the check about appending is made in a separate method and the result is used to append or not. E.G.

if (addUserEmail(userRole,siteId)) {
    allSiteOwners.append(",").append(userEmail);
}